src/main/java/org/jitsi/jicofo/MeetExtensionsHandler.java

```diff
@@ -131,26 +131,13 @@ public void processPacket(Packet packet)
             return;
         }
 
-        if (packet instanceof ColibriConferenceIQ)
-        {
-            handleColibriIq((ColibriConferenceIQ) packet);
-        }
-        else if (packet instanceof MuteIq)
-        {
-            handleMuteIq((MuteIq) packet);
-        }
-        else if (packet instanceof RayoIqProvider.DialIq)
-        {
-            handleRayoIQ((RayoIqProvider.DialIq) packet);
```

Comment:
From all this methods only handleRayoIQ is blocking, but I guess it will not cause any issues to move all of them. But it will only create many thread unnecessary.